### PR TITLE
Update amendments worked example

### PR DESCRIPTION
**Pull request by romifz**

### CODE REVIEW

**Comment by duncandewhurst** [comment]
@jpmckinney do you know why the docs build is breaking?

It'll be easier to review if the built version of the example is available too

**Comment by jpmckinney** [comment]
I forgot to merge an earlier branch that updated the version of commonmark we're using. It should build now.

**Comment by jpmckinney** [comment]
I also had to run `ocdskit indent -r standard/docs/en/examples`

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: I think it would be better to thread the sample JSON files in between each step, as with the [easy releases example](https://standard.open-contracting.org/docs-easy-releases/en/implementation/easy-releases/) and use expanding/collapsing to highlight the relevant sections.

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: depending of jurisdiction, both changes require an official approval -> depending on jurisdiction, certain changes can need to be disclosed as amendment

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
* **The amendments block does not contain data on what was changed**. Changes are recorded by updating the properties of the `tender` block in a new release. 
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: Add quotes to tenderAmendment

Comment (duncandewhurst) [reply standard/docs/en/implementation/amendments.md]: > but can be a convenient way of publishers making structured information on the before and after values of a field more accessible to users. 

Unless I misunderstood, a amendment release should not contain the before values?

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
### Example 1: Tender updates and amendments
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
### Example 2: Contract amendment
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: Please update `versioned-tender.json` to reflect the changes in `merge-tender-3.json`

> * **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section, to see where changes are explained by an amendment `rationale`. 

What information from the amendments array would be looked up in the versionedRelease?

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
This example shows an update to the value and scope of a contract.
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: Explain that a 'contractUpdate' release is published.

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Thanks Romina, updates are looking good!

I made some suggestions in `amendments.md`, also:

1. Is there a reason for having separate merged and versioned files for each example, can we just have one record for each example, which has both the compiled and versioned releases?

2. I found it helpful to generate a record with embedded releases for each example and to review them in OCDS-Show, in order to easily see what had changed between each release. Can you add the embedded releases to the records for each example and include an admonition directing the reader to review the examples in OCDS-Show via the Data Review Tool (e.g https://standard.open-contracting.org/review/?source_url=). Currently amendments are missing from the OCDS-Show interface, but I made a PR to correct that!

3. Can we add an extra example explaining how to preserve detail of individual amendments in an implementation which uses the easy releases model?

e.g. in Australia the source system does not store a change history of individual contract notices, however contract amendments are stored as separate notices, therefore even in the easy releases model it is possible to generate an OCDS release for the contract notice (which is kept up to date with the latest value of the contract) **and** an OCDS release for each amendment to the contract notice.

The advantage of this approach (over just maintaining a single release and updating it's properties and release ID each time there is an amendment) is that users have access to the details of each amendment, i.e. the individual changes to contract value or duration, rather than just the state after the most recent amendment.

There is a worked example already documented for this on [Gist](https://gist.github.com/mrshll1001/6ef17e6990587b8da13114a1050257c2) and the notices the example is based on are available in the AusTender front end ([contract notice](https://www.tenders.gov.au/Cn/Show/6dcfcc43-c5cf-7a3e-30ae-7561f054e523), [amendment notice](https://www.tenders.gov.au/Cn/Show/48661147-ac3c-b9b4-7228-6b852602425e))

File: standard/docs/en/implementation/amendments.md
Comment (romifz) [reply standard/docs/en/implementation/amendments.md]: This is present in the current text. I don't think it means to say the amendment release contains the before values, but that it might be easier to users to have full blocks of data to compare. I'm not sure this is true though, so I will change the sentence.

File: standard/docs/en/implementation/amendments.md
Comment (romifz) [reply standard/docs/en/implementation/amendments.md]: This was in the original example as well. `releaseID` and `amendsReleaseID` can be used to find changes in the `versionedRelease`, I'll clarify this in the text.

Comment (duncandewhurst) [reply standard/docs/en/implementation/amendments.md]: Do you think that it would make more sense to say that `releaseID` and `amendsReleaseID` can be used to find the releases with the before and after values in the `releases` array?

I am wondering which is easier from a user's perspective: checking through every field in the versioned release to find where both `releaseID` and `amendsReleaseID` appear, or comparing the two releases in the releases array.

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [reply standard/docs/en/implementation/amendments.md]: Great, thanks!

**Comment by romifz** [comment]
Hi @duncandewhurst, thanks for the review. I've included your suggestions except for the Data Review Tool link review using the `source_url` param (I couldn't figure how to build the link using markdown, and I'm not sure about having a fixed link in the docs), so I included instructions to download the records instead. I also added more information to the Australia example.

I changed the location of the JSON files we want to make downloadable to `docs/en/_static/json`, I'm not sure if there is a better location.

I wanted to add a link to the probable location of the Easy Releases guidance, but the checks failed. I've put a comment in the docs to add the link once #951 is merged.

PD: I realize I edited some files I shouldn't have in this branch, it is fixed now.

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [reply standard/docs/en/implementation/amendments.md]: Outstanding typo:

'depending of jurisdiction' > 'depending on jurisdiction'

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [reply standard/docs/en/implementation/amendments.md]: Did you have a view on this?

If we don't know which is easier, we can just update the sentence to mention both, e.g.

> * **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section or in other releases, to see where changes are explained by an amendment `rationale`.

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
The Easy releases guidance (<mark>add link</mark>) explains how to publish releases without storing or publishing a full change history. Depending on the source system, it may still be possible to publish a history of amendments when using this model.
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]:
```suggestion
Where the source system stores a history of contract amendments, either as separate notices or as properties of the original contract notice, contract amendments can be published as separate releases in OCDS.
```

File: standard/docs/en/implementation/amendments.md
Comment (duncandewhurst) [inline standard/docs/en/implementation/amendments.md]: I think this is now covered by the suggested change above, but it would be good to replace it with a link to a real life example.

[CN3350299](https://www.tenders.gov.au/Cn/Show/03a3c53e-b3bd-eac1-558a-4c659e44a516) from AusTender has many amendments.

We can add sentence to make it clear that if this was published using only a single release in OCDS, then only the latest contract value and duration would be available to users, however if each amendment is published as a separate release then the full history of changes to the value and duration is available.

**Comment by duncandewhurst** [comment]
Thanks Romina! Looks good - I added a couple of comments to the Australia example and commented on a couple of outstanding bits from the initial review. Once those are all resolved then I'm happy for James to do a final review.

**Comment by duncandewhurst** [comment]
@romifz is this ready to review again? Thanks

**Comment by romifz** [comment]
@duncandewhurst yes it is!

**Review by duncandewhurst — APPROVED** [review]

**Comment by duncandewhurst** [comment]
Thanks Romina, a few comments/typos were unresolved so I've added commits to fix those.

Ready for @jpmckinney to review.

**Comment by jpmckinney** [comment]
Okay to merge, but blocked by the presence of `<mark>add link</mark>` while waiting for #951.

**Pull request merged**
